projects/netgrif-components-core/src/lib/task/services/task-data.service.ts

```diff
@@ -239,7 +239,7 @@ export class TaskDataService extends TaskHandlingService implements OnDestroy {
      * @param nextEvent indicates to the event queue that the next event can be processed
      */
     protected processErroneousGetDataRequest(gottenTaskId: string,
-                                             error: HttpErrorResponse,
+                                             error: any,
```

Comment:
why `any`? If the newly added type is `Error` then the type can be extended into a union `HttpErrorResponse | Error`

Reply:
Done